startpage_content.md

```diff
@@ -0,0 +1,62 @@
+This WIP PR is for preparing content for the start page.
+
+Please commit your ideas until 2019-05-11 so that we can discuss on the next web meeting about the design.
+
+Solutons could be:
+
+* screens (can take a lot of content)
+* carousel/slides (all at top, do not take much content)
+* carousel/slides with screens (teaser + _learn more_-button)
+* ...
+
+# Persistent content
+
+## Green Open Access
+_(Citation advantage of OA https://sparceurope.org/what-we-do/open-access/sparc-europe-open-access-resources/open-access-citation-advantage-service-oaca/ )_
+
+Many researchers do not use their right to make their papers freely available online, in addition to the paywalled version offered by traditional publishers.
+
+There are two major advantages of unpaywalling your papers:
+1. There are a lot of studies giving evidence, that freely available publications have a higher citation rate. 
+This is called the _Open Access Citation Advantage_. You can find a list of studies on https://sparceurope.org/what-we-do/open-access/sparc-europe-open-access-resources/open-access-citation-advantage-service-oaca/oaca-list/ 
+As a conclusion: Your research is not only more acknowledged, but pushes scientific advantage in general, because your research better spreads!
+1. Libraries do not need to by overpriced electronic subscriptions to journals, when they can afford them at all.
+
+## Open repositories
+Uploading your papers on your own webpage is not enough. Such copies are less stable and harder to find than documents uploaded to well-indexed repositories.
+There are for a lot of major repositories like arXiv, chemRxiv and institutional repositories. Nearly all of them are searched by BASE and CORE. And they ensure a long-term availability of your research.
+
+Dissemin searches for copies of your papers in a large collection of open repositories and tells you which ones cannot be accessed. 
```

Comment:
I'd rather say "... which ones it has not been able to access." In my opinion this should be the first sentence.

Then could follow something like: "Our repositories are those indexed by CORE and BASE" (with links to these services), including arXiv, chemRxiv, and institutional repositories such as HAL and ...".

And only then do we explain that we don't take personal web pages into account. (Maybe we could also say something against academic social networks?)